app/predacons_cli/src/rag.py

```
@@ -0,0 +1,200 @@
+
+                    # Parse the content using BeautifulSoup
+                    soup = BeautifulSoup(response.text, 'html.parser')
+
+                    # Extract text from the website
+                    text = ' '.join([p.get_text() for p in soup.find_all('p')])
+
+                    # Append the extracted text to results
+                    results.append(text)
+                    i = i+1
+                    print(url)
+                except requests.RequestException as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)